exercises/practice/resistor-color/src/test/java/ResistorColorTest.java

```diff
@@ -16,36 +14,26 @@ public void setup() {
 
     @Test
     public void testBlackColorCode() {
-        String input = "black";
-        int expected = 0;
-
-        assertEquals(expected, resistorColor.colorCode(input));
+        assertThat(resistorColor.colorCode("black")).isEqualTo(0);
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testWhiteColorCode() {
-        String input = "white";
-        int expected = 9;
-
-        assertEquals(expected, resistorColor.colorCode(input));
+        assertThat(resistorColor.colorCode("white")).isEqualTo(9);
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testOrangeColorCode() {
-        String input = "orange";
-        int expected = 3;
-
-        assertEquals(expected, resistorColor.colorCode(input));
+        assertThat(resistorColor.colorCode("orange")).isEqualTo(3);
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testColors() {
-        String[] expected = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
-
-        assertArrayEquals(expected, resistorColor.colors());
+        assertThat(resistorColor.colors()).isEqualTo(
```

Comment:
`isEqualTo` is probably the wrong assertion for an array. We want `containsExactly` behavior.

Reply:
Good catch.  I'll fix it.